### DISCUSSION TITLE
Remove the "gtm4wp.com" credits/comment from the website?

### DISCUSSION THREAD

**Discussion by danielmat3** [duracelltomi/gtm4wp#336] [Q&A]
Hey there! How I can remove the "gtm4wp.com" credits/comment from the website? Thanks a bunch!

This is what I am talking about:

![credits](https://github.com/duracelltomi/gtm4wp/assets/162363136/8886ee02-ed91-4eab-9602-d66507c6d9be)

**Comment by duracelltomi** [comment]
Since this is a free project, I am sure you understand that this small reference is probably a good way to say thank you.
If you would like to remove it, you can write a PHP code that catches output of the HTML header and does a search/replace in the code.
